.github/workflows/cla.yml

```diff
@@ -18,14 +18,11 @@ jobs:
           PERSONAL_ACCESS_TOKEN : ${{ secrets.CLA_ACCESS_TOKEN }}
         with:
           path-to-signatures: 'CLA/signatures/v1.0/cla.json'
-          path-to-document: 'https://github.com/corgibytes/freshli-cli/blob/main/CLA/CLA-v1.0.md'
+          path-to-document: 'https://github.com/corgibytes/freshli/blob/main/CLA/CLA-v1.0.md'
```

Comment:
No such file exists at this location. Do we need to add it there?

Reply:
I think it would be best for the CLA document to be located in the same repository as the signatures.

Reply:
I agree. I copied this configuration from the Freshli Lib and didn't check the link. I'll fix it there as well. :-)